lib/utils/import.js

```diff
@@ -49,3 +50,32 @@ function getSourceModuleName(node) {
     return undefined;
   }
 }
+
+/**
+ * Gets an import identifier (either imported or local name) from the specified ImportDeclaration.
+ *
+ * @param {node} node the ImportDeclaration to find the import identifier for
+ * @param {*} source the source, or module name string, of the import
+ * @param {*} importName the imported name to find (will return the alias of the import, of found)
```

Comment:
This should say that the parameter is optional and will look for the default import if absent.

Reply:
I'd actually prefer that the API doesn't do that. I think it's much clearer if you always request the `importName`, regardless of whether it's a default or named. Thoughts?

Reply:
The default import can be locally named anything though so it doesn't make sense to me why we should pass an arbitrary string? See these examples:

```
// import EmberComponent from '@ember/component';
getImportIdentifier(node, '@ember/component') // returns 'Component'
```
```
// import Component from '@ember/component';
getImportIdentifier(node, '@ember/component') // returns 'EmberComponent'
```

Reply:
Fair. I do think it's nice to keep consistency for clarity. I may consider renaming the parameter to `namedImportIdentifier` then, so as to disambiguate to folks that this is only used for named imports.